ugvc/mrd/featuremap_consensus_utils.py

```diff
@@ -21,36 +21,40 @@
         FeatureMapFields.X_FC2.value,
         FeatureMapFields.MAX_SOFTCLIP_LENGTH.value,
         FeatureMapFields.X_FLAGS.value,
-        # "a3",
-        # "rq"
     ],
-    "string_list_fields": [
-        FeatureMapFields.X_RN.value,
-        FeatureMapFields.X_CIGAR.value,
-        "rq"
-        # FeatureMapFields.IS_FORWARD.value,
-        # FeatureMapFields.IS_DUPLICATE.value,
-        # "tm"
+    "string_list_fields": [FeatureMapFields.X_RN.value, FeatureMapFields.X_CIGAR.value, "rq", "tm"],
+    "boolean_fields": [
+        FeatureMapFields.IS_FORWARD.value,
+        FeatureMapFields.IS_DUPLICATE.value,
     ],
     "fields_to_write_once": [
         FeatureMapFields.READ_COUNT.value,
+        FeatureMapFields.FILTERED_COUNT.value,
         FeatureMapFields.TRINUC_CONTEXT_WITH_ALT.value,
         FeatureMapFields.HMER_CONTEXT_REF.value,
         FeatureMapFields.HMER_CONTEXT_ALT.value,
-        # FeatureMapFields.IS_CYCLE_SKIP.value,
         FeatureMapFields.PREV_1.value,
         FeatureMapFields.PREV_2.value,
         FeatureMapFields.PREV_3.value,
         FeatureMapFields.NEXT_1.value,
         FeatureMapFields.NEXT_2.value,
         FeatureMapFields.NEXT_3.value,
     ],
+    "boolean_fields_to_write_once": [
+        FeatureMapFields.IS_CYCLE_SKIP.value,
+    ],
 }
 
 
+class FeatureMapFilters(Enum):
```

Comment:
That's good, but could you put it in ugvc/mrd/featuremap_utils.py and import it? ugvc/mrd/srsnv_inference_utils.py has similar definitions so it should import the same structure rather than have this duplicated